apps/alpha-test-app/src/components/layout-demo/FloatingButtonDemo.vue

```diff
@@ -0,0 +1,20 @@
+<script lang="ts" setup>
+import chevronRightSmall from "@sit-onyx/icons/chevron-right-small.svg?raw";
+import { OnyxButton } from "sit-onyx";
+
+const isOpen = defineModel<boolean>();
+defineProps<{ isOverlay: boolean }>();
+</script>
+
+<template>
+  <OnyxButton class="floating-button" :icon="chevronRightSmall" @click="isOpen = !isOpen" />
```

Comment:
I wondered why we have this gap in the button
![image](https://github.com/SchwarzIT/onyx/assets/67898185/4520f7d4-97aa-413e-a957-87d60d2ec8b3)

Seems like we have a bug in the button. The label is a required prop but we defined a default so it gets optional. We should either make the label option and put a v-if on the label or make the label required

Reply:
There is a ticket for a dedicated https://github.com/SchwarzIT/onyx/issues/386 icon button

Reply:
Fine for me but we should still remove the default label value (empty string) because this will make the label optional although its required

Reply:
https://github.com/SchwarzIT/onyx/issues/475 I'd say it's out of scope of this PR ^^ so I created a bug tiket for the next sprint :)